src/app/SettingsPressedUseCaseFactory.java

```diff
@@ -0,0 +1,27 @@
+package app;
+
+import interface_adapter.loggedin.settings.SettingsViewModel;
+import interface_adapter.ViewManagerModel;
+import interface_adapter.loggedin.settings.SettingsButtonController;
+import interface_adapter.loggedin.settings.SettingsButtonPresenter;
+import use_case.settings_pressed.SettingsPressedInteractor;
+import use_case.settings_pressed.SettingsPressedInputBoundary;
+import use_case.settings_pressed.SettingsPressedOutputBoundary;
+
+public class SettingsPressedUseCaseFactory {
+
+
+    private SettingsPressedUseCaseFactory() {}
+
+    //private static create(SettingsViewModel viewModel, ViewManagerModel managerModel) {
+
+
+    public static SettingsButtonController createSettingsButtonController(ViewManagerModel viewManager,
```

Comment:
Same as my previous comment. Also, we usually don't use factories in our project for the purpose of solely creating a controller.